### PR TITLE
adding postcss config for custom tachyons css

### PR DESCRIPTION
**Pull request by Jbarget**

### CODE REVIEW

File: client/src/css/_variables.css [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+ --white-50: rgba(255,255,255,.5);
+ --white-40: rgba(255,255,255,.4);
+ --white-30: rgba(255,255,255,.3);
+ --white-20: rgba(255,255,255,.2);
+ --white-10: rgba(255,255,255,.1);
+ --white-05: rgba(255,255,255,.05);
+ --white-025: rgba(255,255,255,.025);
+ --white-0125: rgba(255,255,255,.0125);
+
+  --dark-red:  #e7040f;
```
Comment (mantagen) [inline client/src/css/_variables.css]: do we need these colours?

Comment (Jbarget) [reply client/src/css/_variables.css]: those are the defaults

File: client/public/index.html [hunk 2]
```diff
@@ -11,6 +11,7 @@
     -->
     <link rel="manifest" href="%PUBLIC_URL%/manifest.json">
     <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico">
+    <link rel="stylesheet" href="style.min.css" />
```
Comment (mantagen) [inline client/public/index.html]: this should be "/static/css/style.min.css"

Comment (mantagen) [reply client/public/index.html]: As discussed we'll change this later @Jbarget

**Pull request merged**